### PR TITLE
SIMSBIOHUB-287: User Access Request

### PR DESCRIPTION
**Pull request by al-rosenthal**
## Links to Jira Tickets

[SIMSBIOHUB-287](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-287)

## Description of Changes

- Updated create access request to place email and display name in the correct fields
- Updated dev users that were affected by issue

## Testing Notes

- Create a test BCeID
- Login to SIMS with your test BCeID and request access
- Login as admin user and approve the request
- Edit a project
- Open the Manage Team members autocomplete and see that the new user display name and email are consistent with other seed users

### CODE REVIEW

**Review by curtisupshall — COMMENTED** [review]
Code looks good! I tested this in Openshift and it works as expected.

One minor bug that you may wish to address in this PR: when you submit your Access Request and are sent back to the home page, the home page doesn't show that you have a pending Access Request until you reload the page. We should peform a refresh of the user's pending access requests when they are sent ot the home page.

**Comment by NickPhura** [comment]
nit: can you update the params/doc for the `getAdministrativeActivityStanding` function in `administrative-activity-service.ts`. I think the param still says userIdentifier, rather than userGuid.

File: api/src/repositories/administrative-activity-repository.ts [hunk 1]
```diff
@@ -182,7 +182,7 @@ export class AdministrativeActivityRepository extends BaseRepository {
    * @return {*}  {(Promise<IAdministrativeActivityStanding>)}
    * @memberof AdministrativeActivityRepository
    */
-  async getAdministrativeActivityStanding(userIdentifier: string): Promise<IAdministrativeActivityStanding> {
+  async getAdministrativeActivityStanding(userGUID: string): Promise<IAdministrativeActivityStanding> {
```
Comment (NickPhura) [inline api/src/repositories/administrative-activity-repository.ts:185]: nit: jsdoc still says userIdentifier

Comment (al-rosenthal) [reply api/src/repositories/administrative-activity-repository.ts:185]: oops, updated

**Review by NickPhura — APPROVED** [review]

**Review by curtisupshall — APPROVED** [review]

**Pull request merged**